components/automate-ui/src/app/page-components/services-sidebar/services-sidebar.component.scss

```diff
@@ -106,24 +107,121 @@ chef-alert {
     font-size: 14px;
   }
 
-  .health-line {
-    display: flex;
+  hr.health-info-divider {
+    margin: 16px -1em 16px -1em;
```

Comment:
So I think this semantically is more like a `hr` than a border on a div. But to do it that way means we have to negate the padding on the enclosing element which couples those two things 😬 🤷‍♀